src/tools/perf/lib/ucp_tests.cc

```diff
@@ -504,6 +504,20 @@ class ucp_perf_test_runner {
         ucp_worker_fence(m_perf.ucp.worker);
     }
 
+    void reset_buffers(const void *src, size_t offset, size_t count)
```

Comment:
1. `psn_t src`
2. no need to pass count - it's always one psn number

Reply:
Done with the 2nd commit. Thanks.